great_expectations/core/usage_statistics/schemas.py

```diff
@@ -1212,8 +1212,6 @@
         "event_time",
         "data_context_id",
         "data_context_instance_id",
-        "mac_address",
-        "oss_id",
```

Comment:
These are not mandatory - if a user is using something before v17, they won't have these